css/banners.css

```diff
@@ -106,28 +91,45 @@ html, body{
   text-decoration: none;
 }
 
+.click-area {
+  width: 100%;
+  height: 5em;
+}
+
+.arrow {
+  transition: 0.2s ease all;
+}
+
+.rotate {
+  -moz-transform: rotate(-180deg);
+  -webkit-transform: rotate(-180deg);
+  -ms-transform: rotate(-180deg);
+  -o-transform: rotate(-180deg);
+  transition: 0.2s ease all;
+}
+
 .expand {
-   height: 450px !important;
-   -webkit-transition: height 0.5s ease-in-out;
-    -moz-transition: height 0.5s ease-in-out;
-    -o-transition: height 0.5s ease-in-out;
-    transition: height 0.5s ease-in-out;
+  height: 28em !important;
+  -webkit-transition: height 0.5s ease-in-out;
+  -moz-transition: height 0.5s ease-in-out;
+  -o-transition: height 0.5s ease-in-out;
+  transition: height 0.5s ease-in-out;
 }
 
 .excerpt {
   display: inherit !important;
   visibility: visible;
   transition: visibility 0s, opacity 0.5s linear;
-
 }
 
 .excerpt p{
   text-align: center;
-  font-family: 'EB Garamond', serif;
-  font-size: 16px;
+  font-family: 'Libre Baskerville', sans-serif;
```

Comment:
$font-accent if you intended this to be italic, $font-body otherwise
or leave it if you hate how Old Standard ($font-body) looks